utils/rpms/daos.spec

```diff
@@ -588,6 +589,9 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 # No files in a shim package
 
 %changelog
+* Fri Jun 28 2024 Michael Hennecke <michael.hennecke@intel.com> 2.7.100-2
```

Comment:
The corresponding VERSION change is missing in this file and we should sync the dates between the rpm and debian files.